markdown/preface.markdown

```diff
@@ -164,9 +164,10 @@ scientific computing.
 
 In your Python travels, you may have already heard a few rumblings about which version of Python is better.
 You may have wondered why it's not just the latest version.
+(Spoiler alert: it is.)
 
 At the end of 2008, the Python core developers released Python 3.0 and, as
-Douglas Adams observed tongue-in-cheek about the creation of the Universe in
+Douglas Adams observed about the creation of the Universe in
```

Comment:
I finally realised that a synonym for "said tongue-in-cheek" is "quipped". =P

Also, we don't need the "in the hitchhiker's guide to the galaxy", that can be footnoted. Thoughts on present phrasing?